docs/docsite/rst/shared_snippets/installing_collections_git_repo.txt

```diff
@@ -0,0 +1,84 @@
+You can install a collection in a git repository by providing the URI to the repository instead of a collection name or path to a ``tar.gz`` file. The collection must contain a ``galaxy.yml`` file, which will be used to generate the would-be collection artifact data from the directory. The URI should be prefixed with ``git+`` (or with ``git@`` to use a private repository with ssh authentication) and optionally supports a comma separated version (for example, a commit or tag).
+
+.. warning::
+
+   Use SSH keys instead of embedding credentials into a git URL.
```

Comment:
AFAIR one may put creds (access token) into `~/.netrc` and it'd work just fine.

Reply:
Ok, I can reword that to just say not to embed credentials instead of listing secure options.

Reply:
Well, removing isn't necessary but you could add something like "Make sure to use safe auth options for security reasons. For example, use SSH, netrc or [`http.extraHeader`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpextraHeader)/[`url.<base>.pushInsteadOf`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf) in Git config to prevent your creds from being exposed in logs."

Reply:
Fixed